CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Next Release
+
```

Comment:
Consider mentioning [Keep a Changelog](http://keepachangelog.com) here? Perhaps something like this...

> The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

Reply:
great idea!